File: AndroidBasics/notes/src/withGms/java/pan/alexander/notes/data/database/AppDatabase.java
package pan.alexander.notes.data.database;

import androidx.room.Database;
import androidx.room.RoomDatabase;

import pan.alexander.notes.domain.entities.Note;
import pan.alexander.notes.domain.entities.Trash;

@Database(entities = {Trash.class}, version = 2)
public abstract class AppDatabase extends RoomDatabase {
    public abstract TrashDao getTrashDao();
}

File: Filmrevealer/app/src/main/java/pan/alexander/filmrevealer/database/AppDatabase.kt
package pan.alexander.filmrevealer.database

import androidx.room.Database
import androidx.room.RoomDatabase
import pan.alexander.filmrevealer.domain.entities.Film
import pan.alexander.filmrevealer.domain.entities.FilmDetails

@Database(entities = [Film::class, FilmDetails::class], version = 10)
abstract class AppDatabase : RoomDatabase() {
    abstract fun filmDao(): FilmDao
    abstract fun filmDetailsDao(): FilmDetailsDao
}
